app/(storyblok)/editor/layout.tsx

```diff
@@ -0,0 +1,13 @@
+import StoryblokProvider from '@/components/StoryblokProvider';
+
+type StoryblokLayoutProps = {
+  children: React.ReactNode,
+};
+
+export default function StoryblokLayout({ children }: StoryblokLayoutProps) {
+  return (
+    <StoryblokProvider isEditor={true}>
```

Comment:
New layout with the isEditor flag to trigger pulling the AccessToken from the URL.